paasta_tools/cli/cmds/spark_run.py

```diff
@@ -23,6 +23,7 @@
 from service_configuration_lib.spark_config import get_spark_conf
 from service_configuration_lib.spark_config import send_and_calculate_resources_cost
 
+from paasta_tools import kubernetes_tools
```

Comment:
super-nit: `from paasta_tools.kubernetes_tool import limit_size_with_hash` is more in-line with how we generally do imports

Reply:
Oh sure!